src/components/TracklistsSection/TracklistsSection.js

```diff
@@ -6,52 +6,43 @@ import Section from '../Section/Section'
 import SectionTitle from '../SectionTitle/SectionTitle'
 import SectionDescription from '../SectionDescription/SectionDescription'
 import TracklistsCard from '../TracklistsCard/TracklistsCard'
-import styles from './TracklistsSection.module.scss'
 
 const TracklistsSection = ({ data }) => {
   return (
-    <Section className="has-background-success">
-      <div
-        className={classnames(styles.container, 'container has-text-primary')}
-      >
-        <SectionTitle title="Tracklists" />
-        <SectionDescription
-          text="Placeat facere possimus, omnis voluptas assumenda est, omnis dolor
+    <div className="has-background-success">
```

Comment:
why should there be a div around section

Reply:
Good point, Section component should have <div> element with section styling instead of <section>, will fix it